docs/csharp/programming-guide/statements-expressions-operators/toc.yml

```diff
@@ -5,38 +5,35 @@
     href: statements.md
   - name: Expressions
     href: expressions.md
-    items:
-    - name: Expression-bodied members
-      href: expression-bodied-members.md
+  - name: Expression-bodied members
+    href: expression-bodied-members.md
   - name: Operators
     href: operators.md
   - name: Anonymous Functions
-    href: anonymous-functions.md
     items:
+    - name: Overview
+      href: anonymous-functions.md
     - name: Lambda Expressions
       href: lambda-expressions.md
-      items:
-      - name: "How to: Use Lambda Expressions in a Query"
-        href: how-to-use-lambda-expressions-in-a-query.md
-      - name: "How to: Use Lambda Expressions Outside LINQ"
-        href: how-to-use-lambda-expressions-outside-linq.md
+    - name: "How to: Use Lambda Expressions in a Query"
+      href: how-to-use-lambda-expressions-in-a-query.md
+    - name: "How to: Use Lambda Expressions Outside LINQ"
+      href: how-to-use-lambda-expressions-outside-linq.md
     - name: Anonymous Methods
       href: anonymous-methods.md
   - name: Overloadable Operators
     href: overloadable-operators.md
   - name: Conversion Operators
     href: conversion-operators.md
-    items:
-    - name: Using Conversion Operators
-      href: using-conversion-operators.md
-    - name: "How to: Implement User-Defined Conversions Between Structs"
-      href: how-to-implement-user-defined-conversions-between-structs.md
+  - name: Using Conversion Operators
+    href: using-conversion-operators.md
+  - name: "How to: Implement User-Defined Conversions Between Structs"
+    href: how-to-implement-user-defined-conversions-between-structs.md
   - name: default value expressions
     href: default-value-expressions.md
   - name: Equality Comparisons
```

Comment:
I think that I would define a note named "Equality and equality comparisons" and have the three topics (Equality comparisons and the two how to topics) as its children.